src/Symfony/Component/Notifier/Bridge/MsTeams/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+5.3.0
+-----
+
+ * Added the bridge
```

Comment:
```suggestion
5.3
---

 * Add the bridge
```

Fabien documented how to write `CHANGELOG.md` files here, please keep a look and adjust it accordingly.

https://github.com/symfony/symfony-docs/pull/14830

Thanks 🙏